pl.c

```diff
@@ -655,8 +655,8 @@ struct track_info *pl_play_selected_row(void)
 			struct simple_track *track = &st->simple_track;
 			rv = pl_play_track(pl_visible, track);
 		}
-		window_goto_top(pl_editable_shared.win);
 	}
+	pl_select_playing_track();
```

Comment:
This seems to have caused a regression. Once a song is playing, it is now impossible to select a new song with <enter>. It will select the song already playing and restart it from the beginning.

Reply:
I'm not sure I understand what you mean. `.` is bound to seek by default.

Reply:
Sorry, I made a mistake with the markup, I meant the `<enter>` key.

Reply:
> I removed the line and haven't noticed any ill effect so far.

That line was to address @flyingmutant's feedback in the previous review https://github.com/cmus/cmus/pull/1008/commits/8fdd16a5013eb56922f531ad0608e78d4f073be3#r541906848. It's partly my fault since I didn't test the changed version before my second approval.

I'll look into this and see what I can do to fix it. I'll also check how it interacts with the `follow` option.

Reply:
FWIW, I have the `follow` option active and it still works in both sequential and shuffle mode.

Reply:
See #1044.